ui/app/AppLayouts/Chat/popups/community/CreateCommunityPopup.qml

```diff
@@ -21,6 +21,7 @@ StatusModal {
     property var store
     property bool isEdit: false
     property QtObject community: popup.store.chatsModelInst.communities.activeCommunity
+    property var onSave: () => {}
```

Comment:
You could use a `signal` instead. It's a little cleaner

Reply:
Discussed offline, couldn't make it work as we want :(

Reply:
I have the same observation, could you in a brief explain please?